cupy_backends/cuda/libs/cusparselt.pyx

```diff
@@ -280,12 +281,18 @@ cpdef init(Handle handle):
     status = cusparseLtInit(<cusparseLtHandle_t*> handle._ptr)
     check_status(status)
 
-
 cpdef destroy(Handle handle):
     """Releases hardware resources used by the cuSPARSELt library"""
     status = cusparseLtDestroy(<cusparseLtHandle_t*> handle._ptr)
     check_status(status)
 
+cpdef getVersion(Handle handle, size_t version):
+    """Get the version number of the cuSPARSELt library"""
+    status = cusparseLtGetVersion(<cusparseLtHandle_t*> handle._ptr,
+                                  <int*> version)
+    check_status(status)
```

Comment:
Our convention for this kind of functions in wrappers is to return the int value directly.

c.f. https://github.com/cupy/cupy/blob/1794c835a086dda4cfb82d1152a495b61a82cbdc/cupy_backends/cuda/api/runtime.pyx#L159-L166

```suggestion
cpdef int getVersion(Handle handle) except? -1:
    """Get the version number of the cuSPARSELt library"""
    cdef int version
    status = cusparseLtGetVersion(<cusparseLtHandle_t*> handle._ptr,
                                  &version)
    check_status(status)
    return version
```

Reply:
```
>>> h =　cupy_backends.cuda.libs.cusparselt.Handle()
>>> cupy_backends.cuda.libs.cusparselt.init(h)
>>> cupy_backends.cuda.libs.cusparselt.getVersion(h)
701
```